### DISCUSSION TITLE
Failed finding new element for particle    418 in element    420;how can i slove it?

### DISCUSSION THREAD

**Discussion by jadeinsnow** [Anura3D/Anura3D_OpenSource#35] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114041575/191401343-c8d98937-64ba-478d-9df7-813bd62cb6e1.png)

**Comment by murphy-john** [comment, marked as answer]
Hello, sorry for the slow reply.

Will you check out [this link](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29#discussion-4274028) about posting discussions? Following the guide will help you find answers quicker by guiding you to places it might already be answered and will help up understand the fully scope of your issue by provided the details about your model. 

Your question was asked [here](https://github.com/Anura3D/Anura3D_OpenSource/discussions/1#discussioncomment-2606839).

**Comment by jadeinsnow** [comment]
Thanks for the reply！ I will check out it!
--------------原始邮件--------------
发件人："John&nbsp;Murphy ***@***.***&gt;;
发送时间：2022年9月30日(星期五) 凌晨1:52
收件人："Anura3D/Anura3D_OpenSource" ***@***.***&gt;;
抄送："jadeinsnow ***@***.***&gt;;"Author ***@***.***&gt;;
主题：Re: [Anura3D/Anura3D_OpenSource] Failed finding new element for particle    418 in element    420;how can i slove it? (Discussion #35)
-----------------------------------

 

 
Hello, sorry for the slow reply.
 
Will you check out this link about posting discussions? Following the guide will help you find answers quicker by guiding you to places it might already be answered and will help up understand the fully scope of your issue by provided the details about your model.
 
Your question was asked here.
